pkg/testing/runner/runner.go

```diff
@@ -447,10 +447,20 @@ func (r *Runner) validate() error {
 
 // getBuilds returns the build for the batch.
 func (r *Runner) getBuilds(b OSBatch) []Build {
-	builds := []Build{}
+	var builds []Build
 	formats := []string{"targz", "zip", "rpm", "deb"}
```

Comment:
Since we started using `slice.Contains` on this, why not define a global `map[string]struct{}` and just look strings up instead? It's way more efficient than iterating through a slice on each loop iteration.